ontology/uco/observable/observable.ttl

```diff
@@ -13103,6 +13129,13 @@ observable:totalSpace
 	rdfs:range xsd:integer ;
 	.
 
+observable:totalStorageCapacityInBytes
+	a owl:DatatypeProperty ;
+	rdfs:label "totalStorageCapacityInBytes"@en ;
+	rdfs:comment "The number of bytes that can be stored on a storage device."@en ;
```

Comment:
Please double-check the CP comment thread.  We added "total" for a reason, and I forget if there was a remark in the CP for adjusting the definition comment.